### PR TITLE
complete fellows server route

### PR DESCRIPTION
**Pull request by joseph-p-pasaoa**
## Description
completed fellows server route
finished addFellow endpoint
added updateFellow endpoint
finished deleteFellow endpoint
updated README with all fellow endpoints

## Motivation and Context
because fellows need to be added, edited, and deleted, too

## How Has This Been Tested?
Postman

## Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: backend/routes/fellows.js [hunk 1]
```diff
@@ -60,93 +60,87 @@ const readFellowByEmail = async (req, res, next) => {
...
-//     res.json({
-//         status: "success",
-//         message: `fellow.${fellowId} data updated`,
-//         payload: response
-//     });
-//   } catch (err) {
-//     handleError(err, req, res, next);
-//   }
-// };
+const createFellow = async (req, res, next) => {
```
Comment (AminesCodes) [inline backend/routes/fellows.js:63]: No need to this function/route here 
This has beed addressed into routes/auth.js

File: backend/routes/fellows.js [hunk 2]
```diff
@@ -60,93 +60,87 @@ const readFellowByEmail = async (req, res, next) => {
...
+        status: "success",
+        message: `new fellow '${fFirstName} ${fLastName}' added`,
+        payload: response
+    });
+  } catch (err) {
+    handleError(err, req, res, next);
+  }
+};
 
+const updateFellow = async (req, res, next) => {
```
Comment (AminesCodes) [inline backend/routes/fellows.js:87]: Same thing for update fellow, it has beed addressed in routes/auth.js

File: backend/routes/fellows.js [hunk 3]
```diff
@@ -60,93 +60,87 @@ const readFellowByEmail = async (req, res, next) => {
...
-f_last_name VARCHAR (30) NOT NULL,
-// f_email VARCHAR (50) REFERENCES users_data(user_email),
-f_picture VARCHAR,
-f_bio VARCHAR,
-f_linkedin VARCHAR (150),
-f_github VARCHAR (150),
-cohort INT REFERENCES classes(class_id),
-want_mentor BOOLEAN NOT NULL DEFAULT FALSE
-*/
+const deleteFellow = async (req, res, next) => {
```
Comment (AminesCodes) [inline backend/routes/fellows.js:121]: Has been implemented in routes/auth.js

File: backend/queries/fellows.js [hunk 4]
```diff
@@ -51,54 +43,61 @@ const getFellowByEmail = async (fEmail) => {  // function name breaks convention
...
     INSERT INTO fellows (
         f_first_name,
         f_last_name,
         f_email,
         cohort_id
     ) VALUES (
-        $/firstName/,
-        $/lastName/,
-        $/email/,
+        $/fFirstName/,
```
Comment (AminesCodes) [inline backend/queries/fellows.js:58]: Can you please put it back the way it was, for consistency on routes/auth.js

File: backend/queries/fellows.js [hunk 5]
```diff
@@ -51,54 +43,61 @@ const getFellowByEmail = async (fEmail) => {  // function name breaks convention
...
     UPDATE fellows
-    SET 
-      f_first_name = $/firstName/,
-      f_last_name = $/lastName/,
-      f_picture = $/picture/,
-      f_bio = $/bio/,
-      f_linkedin = $/linkedIn/,
-      f_github = $/github/,
-      cohort_id = $/cohortId/,
+    SET
```
Comment (AminesCodes) [inline backend/queries/fellows.js:80]: Can you please put it back the way it was, for consistency on routes/auth.js

File: backend/queries/fellows.js (hunk 4 above)
Comment (joseph-p-pasaoa) [reply backend/queries/fellows.js:58]: reverted user object var names

File: backend/queries/fellows.js (hunk 5 above)
Comment (joseph-p-pasaoa) [reply backend/queries/fellows.js:80]: reverted user object var names

File: backend/routes/fellows.js (hunk 1 above)
Comment (joseph-p-pasaoa) [reply backend/routes/fellows.js:63]: createFellow block in routes deactivated

File: backend/routes/fellows.js (hunk 2 above)
Comment (joseph-p-pasaoa) [reply backend/routes/fellows.js:87]: updateFellow block in routes deactivated

File: backend/routes/fellows.js (hunk 3 above)
Comment (joseph-p-pasaoa) [reply backend/routes/fellows.js:121]: deleteFellow block in routes deactivated

**Pull request merged**